LeaderboardBackend.Test/Features/Users/AccountConfirmationTests.cs

```diff
@@ -151,4 +153,191 @@ public async Task ResendConfirmation_Success()
         confirmation.UsedAt.Should().BeNull();
         Instant.Subtract(confirmation.ExpiresAt, confirmation.CreatedAt).Should().Be(Duration.FromHours(1));
     }
+
```

Comment:
Need to split these tests to a new file.

Once that's done, the `HttpClient` creation can be written only once in the file's `[SetUp]` method.
You can store the `FakeClock` instance in a field and use it in the tests to change the time when you need to (`clock.Reset(instant)`).